### ISSUE TITLE
Upstream cancellation of rme

### ISSUE THREAD

**Issue by njakes** [arturolinares/sn-rme#39]
@arturolinares In light of the upstream cancellation of the Rich Markdown Editor, will this project be continued?  I notice that you made a release today (01-19-22).
I certainly hope you are willing to carry it on. This editor is one of the best available and far ahead of what standardnotes currently provides themselves.

**Comment by arturolinares** [comment]
Yes, this editor is the main one I use. I don't have a lot of time to maintain it, but I don't plan to archive it or delete it.

**Comment by lzambarda** [comment]
Maybe it could be replaced by https://github.com/hackernoon/rich-markdown-editor which is a fork of the same editor but it is also regularly updated?
